queries/c/highlights.scm

```diff
@@ -146,6 +145,8 @@
 
 (primitive_type) @type.builtin
 
+(sized_type_specifier _ @type.builtin type: _)
```

Comment:
What does this correspond to in C code? `int32_t` and the such or is it for the function pointers highlight

Reply:
unsigned/signed

Reply:
ah okay then